data_utils/raw_dataset_loader.py

```diff
@@ -68,18 +68,117 @@ def load_raw_data(self, dataset_path: str, config):
         ) = self.__normalize_dataset(dataset=dataset)
 
         serial_data_name = config["name"]
-        serial_data_path = (
-            os.environ["SERIALIZED_DATA_PATH"]
-            + "/serialized_dataset/"
-            + serial_data_name
-            + ".pkl"
-        )
+        if dataset_type == "total":
+            serial_data_path = (
+                os.environ["SERIALIZED_DATA_PATH"]
+                + "/serialized_dataset/"
+                + serial_data_name
+                + ".pkl"
+            )
+        else:
+            # append for train; test; validation
+            serial_data_path = (
+                os.environ["SERIALIZED_DATA_PATH"]
+                + "/serialized_dataset/"
+                + serial_data_name
+                + "_"
+                + dataset_type
+                + ".pkl"
+            )
 
         with open(serial_data_path, "wb") as f:
             pickle.dump(minmax_node_feature, f)
             pickle.dump(minmax_graph_feature, f)
             pickle.dump(dataset_normalized, f)
 
+    def minmax_update(self, serial_data_name=None, data_types=None):
```

Comment:
As is, I think this function should be `SerializedDataLoader._normalize_dataset. Or we could to move the pickling outside `load_raw_data` and separately call `RawDataLoader._save_serialized_data` which would normalize, then dump the pickles

Reply:
I am not sure I understand your comment. For multiple train/test/validate datasets, we need to gather the minimum and maximum values from all of them and then conduct normalization.

Reply:
@streeve I made changes based on our discussion in 245ee5af5e5823b311d785ce2ef410f3167cba0c.